src/routes/transactions/transactions.controller.ts

```diff
@@ -233,6 +234,7 @@ export class TransactionsController {
   @Get('chains/:chainId/safes/:safeAddress/transactions/history')
   @ApiQuery({ name: 'timezone_offset', required: false, type: String })
   @ApiQuery({ name: 'cursor', required: false, type: String })
+  @ApiQuery({ name: 'timezoneId', required: false, type: String })
```

Comment:
Since, we already have `timezone_offset`, my original intention was to include the word id explicitly for added clarity. However, I'll change it to timezone, as that's the more standard term when discussing dates.

Reply:
Done ✅